common/src/main/java/com/discordsrv/common/config/main/OnlineSyncConfig.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * This file is part of DiscordSRV, licensed under the GPLv3 License
+ * Copyright (c) 2016-2025 Austin "Scarsz" Shapiro, Henri "Vankka" Schubin and DiscordSRV contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.discordsrv.common.config.main;
+
+import com.discordsrv.common.abstraction.sync.enums.SyncDirection;
+import com.discordsrv.common.config.configurate.annotation.Constants;
+import com.discordsrv.common.config.configurate.annotation.Order;
+import com.discordsrv.common.config.main.generic.AbstractSyncConfig;
+import org.spongepowered.configurate.objectmapping.ConfigSerializable;
+import org.spongepowered.configurate.objectmapping.meta.Comment;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@ConfigSerializable
+public class OnlineSyncConfig {
+
+    public List<Entry> getEntries() {
+        List<Entry> entries = new ArrayList<>();
+        for (PairConfig pair : pairs) {
+            entries.add(new Entry(
+                    pair.conditionName,
+                    pair.roleId
+            ));
+        }
+
+        return entries;
+    }
+
+    @Comment("The pairs of case-insensitive condition and Discord role ids\n"
+            + "Bukkit valid conditions: %1, %2, %3, %4\n"
+            + "Fabric valid conditions: %1, %5, %6, %7\n"
+            + "And any other custom dimension names"
+    )
+    @Constants.Comment({"online", "world", "world_nether", "world_the_end", "overworld", "the_nether", "the_end"})
```

Comment:
The Bukkit values may be different based on the `level-name` in the server.properties (albeit that is rarely changed). Custom worlds are also possible. 

The comment as is makes it seem like the provided options are the only possible values